app/models/item.rb

```diff
@@ -3,7 +3,15 @@ class Item < ActiveRecord::Base
 
   validates_presence_of :name, :price, :quantity, :shelf_life_days
 
-  def expires
-    Date.today + shelf_life_days.days
+  scope :valid_items, -> {where 'shelf_life_days >
+                          (current_date - date(created_at))'
+                          }
```

Comment:
`}` at 8, 26 is not aligned with `-> {where 'shelf_life_days >` at 6, 22 or `scope :valid_items, -> {where 'shelf_life_days >` at 6, 2.